modules/telco-core-red-hat-advanced-cluster-management.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-core-red-hat-advanced-cluster-management_{context}"]
+= Red Hat Advanced Cluster Management
+
+New in this release::
+* No reference design updates in this release
+
+Description::
+{rh-rhacm-first} provides Multi Cluster Engine (MCE) installation and ongoing zero-touch provisioning (ZTP) lifecycle management for deployed clusters.
+You manage cluster configuration and upgrades declaratively by applying `Policy` custom resources (CRs) to clusters during maintenance windows.
+
+You apply policies with the {rh-rhacm} policy controller as managed by {cgu-operator-full}.
```

Comment:
The content starting on line 16 does not line up with the paragraph above it, which renders a bit weird.